constants.cs

```diff
@@ -89,7 +89,8 @@ public enum TYPES {
 public enum KINDS {
     VAR       =     TOKENS.VAR,
     PROCEDURE =     TOKENS.PROCEDURE,
-    FUNCTION  =     TOKENS.FUNCTION
+    FUNCTION  =     TOKENS.FUNCTION,
+    PARAMETER
```

Comment:
Where are we actually converting our KINDS to TOKENS anyway, outside of this dict?